### ISSUE TITLE
Greek letters in superscript/subscript

### ISSUE THREAD

**Issue by julianeagu** [suchow/Dissertate#66]
My greek letters don't render when they are in the superscript/subscript inside an equation. Regular greek letters render just fine. 
For example $\beta$ renders but $_\beta$ and $^\beta$ do not.

Has anyone had this problem before?

**Comment by saiyanprince** [comment]
Yes, I do. My quick and dirty solution for this is to use \mathrm{\beta} when greek letters occur in superscript/subscript. I would appreciate it if someone could point to a better solution though.

**Comment by julianeagu** [comment]
So I actually traced this to [http://tex.stackexchange.com/questions/265763/no-superscript-or-subscript-greek-letters-with-eb-garamond-and-mathspec](url). It seems to be a problem with EB Garamond. 
Indeed, if I got into Dissertate.cls and comment out this line `\setmathsfont(Digits,Latin,Greek)[Numbers={Proportional}]{EB Garamond}` the equations display okay, but they are in the default font rather than EB Garamond. 
I haven't been able to successfully update EB Garamond to get rid of this issue, but hopefully this will point people in the right direction when looking for a solution.
